src/Components/Navbar.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import Container from "react-bootstrap/Container";
+import Nav from "react-bootstrap/Nav";
+import Navbar from "react-bootstrap/Navbar";
+import logo from "../assets/logos/logo.png"; 
+import { useState } from "react";
+
+function Menu() {
+  const [expanded, setExpanded] = useState(false);
+
+  return (
+    <Navbar
+      variant={"dark"}
+      expand="lg"
+      className="fixed-top"
+      expanded={expanded}
+      style={{
+        fontFamily: "Share Tech Mono",
+      }}
+    >
+      <Container>
+        <Navbar.Brand href="#home">
+          <img src={logo} alt="test" className="logo" />
+        </Navbar.Brand>
+        <Navbar.Toggle
+          aria-controls="basic-navbar-nav"
+          onClick={() => setExpanded(expanded ? false : "expanded")}
```

Comment:
Let's not mix booleans with strings. 
```suggestion
          onClick={() => setExpanded(expanded ? false : true)}
```

And if you are trying to get a switch done, can be as simple as this:

```
onClick={() => setExpanded(!expanded)
```